pkg/pillar/cmd/zedkube/clusterstatus.go

```diff
@@ -157,6 +157,7 @@ func (z *zedkube) publishKubeConfigStatus() {
 	}
 
 	// publish the cluster status for the kube container
+	log.Noticef("publishKubeConfigStatus: publishing")
```

Comment:
How often does this function run? Is it periodical or only on change?
Wouldn't debug log level be more appropriate here to avoid generating too many info logs?

Reply:
Sure, changed to debug for this.